tests/utils/utils.go

```diff
@@ -287,7 +287,7 @@ func CreateDefaultSignatureAggregatorConfig(
 	)
 	// Construct the config values for each subnet
 	return signatureaggregatorcfg.Config{
-		LogLevel: logging.Info.LowerString(),
+		LogLevel: logging.Debug.LowerString(),
```

Comment:
Undo these changes once we get things working nicely.

Reply:
Are things working nicely now, or do we still need this?

Reply:
Made the change back